monkey/monkey_island/cc/environment/utils.py

```diff
@@ -44,8 +43,60 @@ def _create_secure_directory_windows(path: str):
             windows_permissions.get_security_descriptor_for_owner_only_perms()
         )
         win32file.CreateDirectory(path, security_attributes)
+
+    except Exception as ex:
+        LOG.error(f'Could not create a directory at "{path}": {str(ex)}')
+        raise ex
+
+
+def create_secure_file(path: str):
+    if not os.path.isfile(path):
+        if is_windows_os():
+            _create_secure_file_windows(path)
+        else:
+            _create_secure_file_linux(path)
+
+
+def _create_secure_file_linux(path: str):
+    try:
+        flags = (
+            os.O_RDWR | os.O_CREAT | os.O_EXCL
+        )  # read/write, create new, throw error if file exists
+        mode = 0o700  # read/write/execute permissions to owner
+        os.close(os.open(path, flags, mode))
```

Comment:
Why do we even need the complexity of the flags? Why not simply [Path(path).touch(0o700)](monkey/monkey_island/cc/services/utils)?

Reply:
Nice; changed it.